packages/components/listbox/src/base/listbox-item-base.tsx

```
@@ -89,11 +89,18 @@ interface Props<T extends object = {}> extends Omit<ItemProps<"li", T>, "childre
   classNames?: SlotsToClasses<ListboxItemSlots>;
 }
 
-export type ListboxItemBaseProps<T extends object = {}> = Props<T> &
+export type ListboxItemBaseProps<T extends object = {}> = Omit<Props<T>, "onClick"> &
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)